src/pages/Cart/Cart.js

```diff
@@ -1,6 +1,92 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import ItemChoice from '../../Components/ItemChoice/ItemChoice';
+import EmptyCart from './EmptyCart/EmptyCart';
+import Item from './Item/Item';
 import './Cart.scss';
 
-const Cart = () => {};
+const Cart = () => {
+  const [productList, setProductList] = useState([]);
+  const [totalPrice, setTotalPrice] = useState(0);
+
+  useEffect(() => {
+    fetch('/data/productList.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data);
+      });
+  }, []);
+
+  return (
+    <div className="cart">
+      <h2 className="tit_cart">
+        장바구니 <span className="screen_out">담긴상품 : </span>
+        {productList.length}개
+      </h2>
+      {productList.length === 0 ? (
+        <EmptyCart />
+      ) : (
+        <div className="group_cart">
+          <div className="wrap_listcart">
+            <strong className="screen_out">장바구니에 담긴 상품 목록</strong>
+            <span className="item_choice">
+              <input type="checkbox" id="checkBox" className="inp_check" />
```

Comment:
```suggestion
      {productList.length === 0 ? <EmptyCart /> : <FullCart /> }
```
위와 같이 Cart.js를 보기 좋게 만들면 어떨까요? 하위 내용들은 모두 FullCart로 컴포넌트화 되는 구조입니다.
네이밍은 다시 해주세요!

Reply:
수정했습니다~! 확인 부탁드립니다 감사합니다 :)